app/src/main/res/drawable/gradient.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <gradient
+    android:startColor="#E263BD5C"
```

Comment:
There is a file in `rest -> values` called` colors.xml` that will contain all your color resources. Then, by doing this, you could use this color anywhere. Also, if this color changes for any reason, you would not need to find where it is used in all the files, since you could only modify it in this place.
If you want more information about this: https://developer.android.com/guide/topics/resources/more-resources#Color.

Edit: I see this in many places. Please change it in every place 😅

Reply:
What happened with this correction?